features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher-new/source/src/app/data/APIValidation.js

```diff
@@ -15,13 +15,51 @@
  * specific language governing permissions and limitations
  * under the License.
  */
+
 import Joi from '@hapi/joi';
+import API from '../data/api';
```

Comment:
Shall we use the `AppData` alias here

```suggestion
import API from 'AppData/api';
```

Reply:
Fixed in f20d89587c00c4aadaf582e15f4f469743872571